requirements-test.txt

```diff
@@ -4,3 +4,5 @@ contexttimer
 # QA checks
 openwisp-utils[qa]~=0.7.0
 packaging~=20.4
+coreapi==2.3.3
```

Comment:
coreapi is planned to be deprecated

Reply:
@auvipy great point, thanks for pointing that out! Do you have suggestions about the best way to handle this?